### PR TITLE
Update PPP, fix typo in manual testing and remove toc

### LINKED ISSUE #297: Table of contents to be removed from DG and UG
3. Both **DG** and **UG** (need add {:.toc} to ignore `Table of Contents` title)

image

### LINKED ISSUE #299: Test case and description don't match
image

### PR DESCRIPTION
**Pull request by TaoJun99**
Fixes #299 
Fixes #297 
Fix typo in manual testing section and remove table of contents heading from toc

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1543,7 +1544,7 @@ testers are expected to do more *exploratory* testing.
       persons have the assignment you wish to give using the `show` command.
 
    2. Test case: `giveall m/CS2100 d/Assignment 3 by/ 11/11/2021` <br>
-      Expected: All persons in the module "CS2100" have "Assignment 1" with due date "10/10/2021" added into their assignment list.
+      Expected: All persons in the module "CS2100" have "Assignment 1" with due date "11/10/2021" added into their assignment list.
```
Comment (weichang18) [inline docs/DeveloperGuide.md:1547]: I think you missed out assignment 1 should be 3

**Review by weichang18 — CHANGES_REQUESTED** [review]
Overall it is good, only 1 thing to change

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (TaoJun99) [reply docs/DeveloperGuide.md]: Good catch!

**Review by weichang18 — APPROVED** [review]
LGTM!!

**Pull request merged**